odoo/api/__init__.py

```diff
@@ -0,0 +1,21 @@
+# ruff: noqa: F401
```

Comment:
Add a comment explaining why this is `api/__init__.py` and not `api.py`: to ease forward-ports. Do the same for the other modules. Maybe add a "TODO Odoo 22: rename `api/__init__.py` => `api.py`"